README.md

```diff
@@ -24,6 +24,9 @@ should be monitored by one exporter.
 
 The installation of the connector is covered in the `Installation` section.
 
+3. Some metrics are collected on the HANA monitoring views by the [SAP Host agent](https://launchpad.support.sap.com/#/notes/2101350). Make sure to have it installed and running to have access to all the monitoring metrics.
```

Comment:
I don't think imho we should put the link because, for example that link requires a login and User which I don't have.

Reply:
I was hesitating about that and I asked @diegoakechi , he told me it is a normal thing for SAP users.

Reply:
@diegoakechi @arbulu89 Imho personally allthough our exporter is SAP centric, having a link where I need to login/create user is quite un-usual in doc. IMho I would just remove it.

and add something public like;
https://help.sap.com/saphelp_nwpi711/helpdata/en/21/98c443122744efae67c0352033691d/frameset.htm or others

Reply:
I agree with having something public.

Reply:
@MalloZup I am fine to switch to the public link.  @juadk can you please do this change?

Reply:
@MalloZup @diegoakechi Done!